2005-06-10  Randolph Chung  <tausq@debian.org>

        * elf/elf.h (R_PARISC_PLABEL21L, R_PARISC_PLABEL14R): Define.

===================================================================
RCS file: /var/lib/cvs/glibc/elf/elf.h,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- elf/elf.h	2005/06/09 01:14:04	1.3
+++ elf/elf.h	2005/06/10 23:41:20	1.4
@@ -1703,6 +1703,8 @@
 #define R_PARISC_LTOFF_FPTR14R	62	/* LT-rel. fct ptr, right 14 bits. */
 #define R_PARISC_FPTR64		64	/* 64 bits function address.  */
 #define R_PARISC_PLABEL32	65	/* 32 bits function address.  */
+#define R_PARISC_PLABEL21L	66	/* Left 21 bits of fct ptr.  */
+#define R_PARISC_PLABEL14R	70	/* Left 21 bits of fct ptr.  */
 #define R_PARISC_PCREL64	72	/* 64 bits PC-rel. address.  */
 #define R_PARISC_PCREL22F	74	/* 22 bits PC-rel. address.  */
 #define R_PARISC_PCREL14WR	75	/* PC-rel. address, right 14 bits.  */

